### PR TITLE
Docker 18.03.0-ce

### PR DESCRIPTION
**Pull request by se35710**
Solves #2219 

Duplicates the [docker-compose.json](https://github.com/lukesampson/scoop/blob/master/bucket/docker-compose.json) and [docker-machine.json](https://github.com/lukesampson/scoop/blob/master/bucket/docker-machine.json) manifests, plus adding Kitematic.

Maybe not the prettiest, but it does it's job.

### CODE REVIEW

File: bucket/docker.json [hunk 1]
```diff
@@ -1,41 +1,39 @@
...
-            "url": "https://get.docker.com/builds/Windows/i386/docker-17.05.0-ce.zip",
-            "hash": "99820746a362466700718d33ee727b9aea993b9c2e05fb2c53ed606e9f6780f1"
+            "url": "https://github.com/docker/toolbox/releases/download/v18.03.0-ce/DockerToolbox-18.03.0-ce.exe",
+            "hash": "7c412e867e4708ec1b36005a217161131a21430a0ef0b0db3591218ec9ec5091"
         }
     },
     "homepage": "https://docker.com",
-    "extract_dir": "docker",
+    "innosetup": true,
+    "post_install": "rm -r $dir\\installers",
```
Comment (rasa) [inline bucket/docker.json]: Change
```
   "post_install": "rm -r $dir\\installers",
```
To
```
   "post_install": "rm -r \"$dir\\installers\"",
```

File: bucket/docker.json [hunk 2]
```diff
@@ -1,41 +1,39 @@
...
+        "docker-compose.exe",
+        "docker-machine.exe"
+    ],
+    "shortcuts": [
+        [
+            "kitematic\\Kitematic.exe",
+            "Kitematic"
+        ]
     ],
     "checkver": {
```
Comment (rasa) [inline bucket/docker.json]: This can be simplified to:
```
"github": "https://github.com/docker/toolbox"
```
right?

Comment (rasa) [inline bucket/docker.json]: This can be simplified to:
```
"github": "https://github.com/docker/toolbox"
```
right?

Comment (se35710) [reply bucket/docker.json]: @rasa Yes, correct! Thanks.

File: bucket/docker.json [hunk 3]
```diff
@@ -22,7 +22,7 @@
         ]
     ],
     "checkver": {
-        "url": "https://github.com/docker/toolbox/releases/latest",
+        "github": "https://github.com/docker/toolbox/releases/latest",
```
Comment (rasa) [inline bucket/docker.json]: Please remove the `/releases/latest` and the `re` line, as they are unneeded. Thx.

Comment (se35710) [reply bucket/docker.json]: re is needed due to the `v` in the release tag, the /releases/latest was removed with the [commit](https://github.com/lukesampson/scoop/pull/2235/commits/3a2b6fae07f9d16887c4200bde256008b6305a3c) after.

Comment (rasa) [reply bucket/docker.json]: `re` is not needed, as checkver already has that regex. See https://github.com/lukesampson/scoop/blob/master/bin/checkver.ps1#L56

Comment (se35710) [reply bucket/docker.json]: Ahh, sorry. it's the `-ce` and possibly `-rc` parts of their version naming that the default regex doesn't catch. I could add `-ce` to the download urls?

Comment (rasa) [reply bucket/docker.json]: You’re right, my bad. Let’s get this puppy merged.

**Pull request merged**